internal/exec/vendor_model.go

```
@@ -0,0 +1,328 @@
+			if p.sourceIsLocalFile {
+				tempDir = path.Join(tempDir, filepath.Base(p.uri))
+			}
+			if err := cp.Copy(p.uri, tempDir, copyOptions); err != nil {
+				return installedPkgMsg{
+					err:  fmt.Errorf("failed to copy package: %w", err),
+					name: p.name,
+				}
+			}
+		default:
+			return installedPkgMsg{
+				err:  fmt.Errorf("unknown package type: %s package", p.pkgType.String(), p.name),
```

golangci-lint

printf: fmt.Errorf call needs 1 arg but has 2 args

(govet)